src/main/java/uk/gov/hmcts/darts/addaudio/validator/AddAudioValidator.java

```diff
@@ -27,15 +34,24 @@ public class AddAudioValidator {
     private boolean validateAddAudio;
     @Value("${darts-gateway.add-audio.fileSizeInMegaBytes}")
     private long expectedFileSize;
+    @Value("${darts-gateway.add-audio.maxFileDuration}")
+    private Duration maxFileDuration;
+
+    @Autowired
+    private final AllowedMediaConfig allowedMediaConfig;
 
     private final XmlWithFileMultiPartRequestHolder multiPartRequestHolder;
     private final XmlValidator xmlValidator;
 
-    public void validate(AddAudio audio) {
-        validateXml(audio);
+    public void validate(AddAudio addAudio) {
+        validateXml(addAudio);
         validateSize();
```

Comment:
does this method load the entire file into memory?

Reply:
as far as i know it uses the multiPartRequestHolder and reads the information from it's stream. @mestebanez can you comment on the use of this? thanks

Reply:
yep all stream based